frontend_tests/node_tests/search_suggestion.js

```diff
@@ -407,7 +407,6 @@ init();
     assert.equal(describe('is:alerted'), 'Alerted messages');
     assert.equal(describe('is:unread'), 'Unread messages');
     assert.equal(describe('sender:bob@zulip.com'), 'Sent by me');
-    assert.equal(describe('stream:devel'), 'Stream <strong>devel</strong>');
```

Comment:
ohk, I saw your commit message later, so can you just provide a bit detail about what was weird?